src/Factory.php

```diff
@@ -204,47 +203,47 @@ public static function toFile($filename, $config)
     /**
      * Set reader plugin manager
      *
-     * @param ReaderPluginManager $readers
+     * @param ContainerInterface $readers
      * @return void
      */
-    public static function setReaderPluginManager(ReaderPluginManager $readers)
+    public static function setReaderPluginManager(ContainerInterface $readers)
     {
         static::$readers = $readers;
     }
 
     /**
      * Get the reader plugin manager
      *
-     * @return ReaderPluginManager
+     * @return ContainerInterface
      */
     public static function getReaderPluginManager()
     {
         if (static::$readers === null) {
-            static::$readers = new ReaderPluginManager(new ServiceManager());
+            static::$readers = new StandaloneReaderPluginManager();
         }
         return static::$readers;
     }
 
     /**
      * Set writer plugin manager
      *
-     * @param WriterPluginManager $writers
+     * @param ContainerInterface $writers
      * @return void
      */
-    public static function setWriterPluginManager(WriterPluginManager $writers)
+    public static function setWriterPluginManager(ContainerInterface $writers)
     {
         static::$writers = $writers;
     }
 
     /**
      * Get the writer plugin manager
      *
-     * @return WriterPluginManager
+     * @return ContainerInterface
```

Comment:
As above, maybe we need to have here `ContainerInterface|StandaloneWriterPluginManager`?